formapp/app.py

```diff
@@ -68,8 +73,15 @@
 flasklog = logging.getLogger('werkzeug')
 flasklog.setLevel(logging.ERROR)
 
-#Display IP address to console for user connection
-print("[info] " +'Running formbar server on:' + ip)
+#Display IP address to console for user connection. Updates the Time as well
+def logFile(type, message):
+    now = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    f = open('log.txt', 'a')
+    f.write("[" + str(now) + "]" + " [" + type + "] " + message + "\n")
+    f.close()
+    print("[" + str(now) + "]" + " [" + type + "] " + message)
+
```

Comment:
Move this function declaration down with the rest of them function declarations.